pyrit/attacks/multi_turn/tree_of_attacks.py

```diff
@@ -211,6 +211,7 @@ def __init__(
         memory_labels: Optional[dict[str, str]] = None,
         parent_id: Optional[str] = None,
         prompt_normalizer: Optional[PromptNormalizer] = None,
+        error_score_map: dict[str, float],
```

Comment:
This should be optional but have a reasonable default. That is 
```suggestion
        error_score_map: Optional[dict[str, float]] = None
```

And if None is passed (default) then we just catch content filter errors and make them 0